packages/tools/devtools/devtools-browser-extension/src/devtools/TelemetryLogging.ts

```diff
@@ -70,12 +71,19 @@ export class OneDSLogger implements ITelemetryBaseLogger {
 	 */
 	private readonly enabled: boolean = false;
 
+	private readonly sessionID?: string;
+	private readonly continuityID?: string;
+	private readonly CONTINUITY_ID_KEY = "myAppBrowserId";
```

Comment:
A bit to follow the naming convention we use for other flags/keys, and to complete the move away from the potentially misleading "browserId".
```suggestion
	private readonly CONTINUITY_ID_KEY = "Fluid.Devtools.ContinuityId";
```